tests/node/test_wakunode_filter.nim

```diff
@@ -25,12 +25,12 @@ import
 
 let FUTURE_TIMEOUT = 1.seconds
 
-suite "Full Node - Waku Filter - End to End":
+suite "Waku Filter - End to End":
   var client {.threadvar.}: WakuNode
   var clientPeerId {.threadvar.}: PeerId
   var server {.threadvar.}: WakuNode
   var serverRemotePeerInfo {.threadvar.}: RemotePeerInfo
-  var pubsubTopic {.threadvar.}: PubsubTopicy
+  var pubsubTopic {.threadvar.}: PubsubTopic
```

Comment:
What?! How this could even compile xD?
That makes me suspect that maybe we are not running this test at all?

Reply:
I arrived at the same thought you did 😂 And I do think, although I'm not sure, that some tests are not ran; will check that later.